src/webextension/widgets/filter-input.js

```diff
@@ -66,7 +66,8 @@ export default class FilterInput extends React.Component {
                onChange={(e) => this.updateValue(e.target.value)}
                ref={(element) => this.inputElement = element}/>
         <Localized id="filter-input-clear">
-          <button type="button" title="cLEAr" disabled={disabled}
+          <button type="button" title="cLEAr"
+                  disabled={disabled ? disabled : !this.state.value}
```

Comment:
`disabled || !this.state.value` is probably simpler. I'm not sure how this would interact with a11y stuff though...

Reply:
>`disabled || !this.state.value` is probably simpler

Oh, duh.. thanks.

>I'm not sure how disabling this (rather than removing the element entirely) would interact with a11y stuff though

Fair. Per the spec, the [disabled attribute](https://www.w3schools.com/tags/att_button_disabled.asp) means the button "is unusable and un-clickable" so feels like its going to be safely ignored anyway. I see that reflected in the fact that 'tabbing' does not try to stop on this hidden element, it goes straight from the input to the item list.